docs/editors-guide/import-terms-for-logical-axioms.md

```diff
@@ -1,80 +1,32 @@
+_Last updated 27-Mar-2024_
 
-### Preferred Instructions
+## Import terms into Mondo for use in logical axioms
 
-_Added 2020-05-06_
+This workflow is for adding classes from external ontologies (i.e., GO or CHEBI), which is much more streamlined compared to [MIREOTing](https://github.com/obophenotype/human-phenotype-ontology/wiki/Editor-Guide#mireoting).
 
-1. Open the `src/ontology/imports/*_terms.txt`
-2. Add term to this file
+1. Fetch the latest changes from `master` in the "mondo" repo
+1. Create a new git feature branch
+1. Open the `src/ontology/imports/manual_seed.txt` file
+2. Add the IRI of the term(s) you want to add to this document and save the file
+    - IRIs for any entity can be added into the `manual_seed.txt` file, e.g. gene, NCBITaxon, etc.
 3. In the Terminal, run: `export "MEMORY_GB=15"`
-4. Then run import: `sh run.sh make refresh-merged`
-4. Close Protege and open again
-5. Edit in Protege
-
----
-#### Detailed instructions for adding a new NCBI Taxon identifier 
-
-_Last updated 6-Mar-2024_
-
-1. Fetch the latest changes from `master`
-1. Create a new feature branch for the updated NCBI Taxon additions
-1. Open the `src/ontology/imports/ncbitaxon_terms.txt` file
-1. Add the IRI(s) of interest into the file, e.g. `http://purl.obolibrary.org/obo/NCBITaxon_3052303`
-1. In the Terminal, run: `export "MEMORY_GB=15"`
-1. Run import: `sh run.sh make refresh-merged` (Note: this may take ~2 hours)
-1. Review and commit the updated files, the changed files will be:
-    - `src/ontology/imports/ncbitaxon_terms.txt`
-    - `src/ontology/imports/merged_import.owl`
-1. Publish the branch and create a PR
-1. Once the PR is approved and merged, the new NCBI Taxon can be referenced in logical definitions
-1. Check if the new NCBI Taxon identifier(s) exist in [https://github.com/obophenotype/ncbitaxon/blob/master/subsets/taxon-subset-ids.txt](https://github.com/obophenotype/ncbitaxon/blob/master/subsets/taxon-subset-ids.txt)
-    - if the identifiers are not in the file, create a PR in this repo to include the new identifiers
-
-    Note: this file contains CURIEs (not IRIs) so the identifier should be added in this format `NCBITaxon:1`
-
-    This additional step is needed since we are not using NCBI Taxon directly, but the OBO slim, and this file is the seed of the NCBITaxon slim.
+4. Then refresh the imports: `sh run.sh make refresh-merged` 
+    - All the imports will be updated, which means that you might see changes in your GitHub diff in the following files:
+        - `src/ontology/imports/*_terms.txt`
+        - `src/ontology/imports/merged_import.owl`
+    - The terms added in the `manual_seed.txt` file will be added to the appropriate import file (e.g human genes will be added to hgnc_terms.txt; NCBITaxon will be added to ncbitaxon_terms.txt) and removed from the `manual_seed.txt` file after the import refresh has run. 
```

Comment:
Why are there currently entries in the `manual_seed.txt` file on [master](https://github.com/monarch-initiative/mondo/blob/master/src/ontology/imports/manual_seed.txt) if these entries are removed after refreshing the imports?

Reply:
Indeed, I dont think that is the case. 

```suggestion
    - The terms added in the `manual_seed.txt` file will be added to the appropriate import file (e.g human genes will be added to hgnc_terms.txt; NCBITaxon will be added to ncbitaxon_terms.txt). After the import is refreshed and the term used in at least one axiom, the term can safely be removed from `manual_seed.txt`. 
```

Reply:
It was not 100% clear the suggestion fits with the curation workflow, so for now it is not included and this last sentence removed from the docs: `After the import is refreshed and the term used in at least one axiom, the term can safely be removed from `manual_seed.txt`. `